src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenConditionalOperatorTests.cs

```diff
@@ -13,6 +13,313 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
 {
     public class CodeGenConditionalOperatorTests : CSharpTestBase
     {
+        [Fact, WorkItem(61483, "https://github.com/dotnet/roslyn/issues/61483")]
+        public void Branchless_Compare()
+        {
+            var source = """
+                using static System.Console;
+                WriteLine(C.Compare(1, 2));
+                WriteLine(C.Compare(3, 3));
+                WriteLine(C.Compare(5, 4));
+
+                class C
+                {
+                    public static int Compare(int x, int y)
+                    {
+                        int tmp1 = (x > y) ? 1 : 0;
```

Comment:
> 1 : 0 [](http://example.com/codeflow?start=44&length=6)

It is probably good to test what happens with other combinations of constants:
- 1 : 1
- 0 : 0
- 2 : 0
- 1 : 2